parm/snow/jcb-prototype_2dvar.yaml.j2

```diff
@@ -0,0 +1,12 @@
+# Algorithm
+# ---------
+algorithm: 3dvar
```

Comment:
2dvar or 3dvar? your script name is 2dvar but I am not sure.

Reply:
there is no 2dvar algorithm in JEDI, it's 3DVar just with a single value in z